content/_index.md

```diff
@@ -65,6 +65,10 @@ section 4. Next steps
 Complete our [Advanced Analytics for Web](https://docs.snowplow.io/accelerators/web/) accelerator if you haven't yet started collecting data from your website.
 {{% /notice %}}
 
+{{% notice tip %}}
```

Comment:
If we can combine the two initial Tips into one, I think it will be a bit clearer and easier on the eyes :)

![image](https://github.com/snowplow/snowplow-ecommerce-tracking-accelerator/assets/15251081/d2032b09-8595-40d3-97f8-a4e4d52ce113)